x-pack/plugins/infra/common/inventory_models/container/metrics/tsvb/container_k8s_overview.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { TSVBMetricModelCreator, TSVBMetricModel } from '../../../types';
+
+export const containerK8sOverview: TSVBMetricModelCreator = (
+  timeField,
+  indexPattern,
+  interval
+): TSVBMetricModel => ({
+  id: 'containerK8sOverview',
+  requires: ['kubernetes.container'],
+  index_pattern: indexPattern,
+  interval,
+  time_field: timeField,
+  type: 'timeseries',
+  series: [
+    {
+      id: 'cpu',
+      split_mode: 'everything',
+      metrics: [
+        {
+          field: 'kubernetes.container.cpu.usage.node.pct',
```

Comment:
Similar here, let's use `kubernetes.container.cpu.usage.limit.pct` instead, it follows the same mode as for memory and is what we use in the infra metric tables.